examples/server/server.c

```diff
@@ -2960,8 +3084,31 @@ THREAD_RETURN WOLFSSL_THREAD server_test(void* args)
 
     #if defined(WOLFSSL_TLS13) && defined(HAVE_SUPPORTED_CURVES)
         if (version >= 4) {
-            SetKeyShare(ssl, onlyKeyShare, useX25519, useX448, usePqc,
-                        pqcAlg);
+            #ifdef CAN_FORCE_CURVE
+            if (force_curve_group_id > 0) {
+                do {
+                    ret = wolfSSL_UseKeyShare(ssl, WOLFSSL_ECC_SECP384R1);
```

Comment:
This seems hard-coded for SECP384R1. Should it be using `force_curve_group_id`?

Reply:
Yes, it should. Oddly it still works: adding this makes it just complain sooner in the UseKeyShare call, like when trying to use SECP224R1 (because bit strength < 256), rather than failing the set_groups function a few lines later. Why, I'm not sure, but I'm firing off another commit right now after testing this locally on P{224|256|384|521|R1 and CURVE{25519|448}. Thanks! I'll send that email.